tensorflow/python/keras/engine/base_layer.py

```diff
@@ -2421,7 +2421,7 @@ def _maybe_cast_inputs(self, inputs, input_list=None):
     compute_dtype_object = self._compute_dtype_object
     should_autocast = (
```

Comment:
what is autocast used for? is this part of mixed precision training?

Reply:
I think it is safer to cast to cus explicitly as opposed to letting keras to do it automatically.

Reply:
> is this part of mixed precision training?
yes

Reply:
> I think it is safer to cast to cus explicitly as opposed to letting keras to do it automatically.

If we manually cast to cus, the logics of loss-scaling is required to be added manually.